variables.tf

```
@@ -1,82 +1,72 @@
 # Copyright (c) Tetrate, Inc 2022 All Rights Reserved.
 
 variable "distribution" {
+  description = "Istio distribution"
```

convention on ordering:
- Description of variable
- Type/structure of variable
- Default value if any

---

```suggestion
  description = "Tetrate Istio Distro"
```
can we please adjust it as the correct commercial name?

---

Updated in https://github.com/tetratelabs/terraform-eksblueprints-tetrate-istio-addon/pull/12/commits/6731e39d82fef17f4675d5b1c0fbb19e8b53ce7a - please let me know if there is anything else to address, thank you! @PetrMc